meta/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
-dependencies: []
+dependencies:
+  - role: elliotweiser.osx-command-line-tools
+    version: 1.1.0
```

Comment:
Locking into a version does help stability somewhat... but also means it's another thing to maintain as the upstream role is improved/fixed. Since there's no concept of a 'major version' I can lock to, I would rather not favor a particular version, but instead keep this role working with the upstream and suggest the latest (e.g. just `- elliotweiser.osx-command-line-tools`).